### PR TITLE
Describe ToT docker image version as head commit short sha

### PR DESCRIPTION
**Pull request by 0405ysj**

### CODE REVIEW

File: .github/workflows/docker.yaml
Comment (0405ysj) [inline .github/workflows/docker.yaml:3]: `pull_request` is for testing this workflow. Should be removed before merging this PR.

**Comment by 0405ysj** [comment]
@ser-io PTAL

**Review by ser-io — CHANGES_REQUESTED** [review]
Let's move these jobs to the Artifacts workflow https://github.com/google/android-cuttlefish/blob/main/.github/workflows/artifacts.yaml, this way our users can find all our artifacts in the same workflow page.

**Comment by 0405ysj** [comment]
> Let's move these jobs to the Artifacts workflow https://github.com/google/android-cuttlefish/blob/main/.github/workflows/artifacts.yaml, this way our users can find all our artifacts in the same workflow page.

I'm not still certain with strategies for managing workflows. Though I move docker images into `artifacts.yaml`, we cannot say all our artifacts are in there. For instance, `host-image-ci.yml` uploads lots of host image artifacts to there.

How do you want to manage Github workflow, in overall?

**Comment by ser-io** [comment]
> > Let's move these jobs to the Artifacts workflow https://github.com/google/android-cuttlefish/blob/main/.github/workflows/artifacts.yaml, this way our users can find all our artifacts in the same workflow page.
> 
> I'm not still certain with strategies for managing workflows. Though I move docker images into `artifacts.yaml`, we cannot say all our artifacts are in there. For instance, `host-image-ci.yml` uploads lots of host image artifacts to there.
> 
> How do you want to manage Github workflow, in overall?

Yes, I missed `host-image-ci.yml` exporting artifacts and didn't act proactively there. I'll work with @WeiChungHsu to have them moved to a single Github workflow, so we can find all of our ToT artifacts on a single page.

**Comment by 0405ysj** [comment]
> Yes, I missed host-image-ci.yml exporting artifacts and didn't act proactively there. I'll work with @WeiChungHsu to have them moved to a single Github workflow, so we can find all of our ToT artifacts on a single page.

Ack. I updated Github workflow, to build artifacts in `.github/workflows/artifacts.yaml`.

**Review by ser-io — APPROVED** [review]

**Review by ikicha — APPROVED** [review]

**Comment by 0405ysj** [comment]
@adelva1984 @rmuthiah @Databean @jmacnak @jemoreira Could anyone approve this PR? According to the list of code owners, I think small fix on README.md located at top dir requires additional reviewers. Thanks!

https://github.com/google/android-cuttlefish/blob/main/.github/CODEOWNERS

**Review by Databean — APPROVED** [review]

**Comment by 0405ysj** [comment]
Thanks!

**Pull request merged**
